lib/active_record/annotate.rb

```diff
@@ -45,6 +45,7 @@ def models
           
           klass = class_name_for(short_path)
           next unless klass < ActiveRecord::Base # collect only AR::Base descendants
+          next if klass.name == "ApplicationRecord" # skip ApplicationRecord
```

Comment:
will this work if my ApplicationRecord is namespaced? Like `MyFancy::Domain::ApplicationRecord`?

Reply:
Not sure but im sure this is a silly PR as it has such a limited reach.

We need some other way to ignore certain files. Maybe we could have an `.annotate_ignore` file in the project root where we list these?

Reply:
I believe that we don't need much configuration, but it would be nice to ignore conventional stuff like `ApplicationRecord`

Also, 

```
irb(main):019:0> Foo::Bar::Quux.name
=> "Foo::Bar::Quux"
```

So we could replace it with:

```
irb(main):020:0> *, name = "Foo::Bar::Quux".split(':')
=> ["Foo", "", "Bar", "", "Quux"]
irb(main):023:0> *, name = "Quux".split(':')
=> ["Quux"]
```

Reply:
Yeah but what happens if someone names an actual model application record then we have an issue. I would prefer something like described in #13

Reply:
I wonder how many people actually see this problem 

I can tell that my project only has two application records at this moment. Both are called `ApplicationRecord`

A simple fix should suffice for a while. Better cover 95% of use cases while we think about custom configuration

Reply:
Ignoring abstract models is much more appropriate for this.

Reply:
Sounds like a plan

Reply:
Checked:

* If there are some annotations, `db:annotte` won't remove them
* If I remove them manually and re-run `db:annotate`, the gem won't add them
* If I rollback to the previous version, `db:annotate` would override my changes and add the error

Works well now

Reply:
Yeah, that's alright. Pretty much expected it.

Thanks!